cmd/test/test.go

```diff
@@ -151,10 +151,13 @@ func newTestCommand() *cobra.Command {
 				return fmt.Errorf("unmarshal parameters: %w", err)
 			}
 
-			outputFilePath, err := cmd.Flags().GetString("file")
+			outputFormats, err := cmd.Flags().GetStringSlice("output")
 			if err != nil {
 				return fmt.Errorf("reading flag: %w", err)
 			}
+			if len(outputFormats) == 0 {
+				outputFormats = []string{output.OutputStandard}
+			}
```

Comment:
@simonbaird @zregvart My latest push should fix acceptance tests. The issue was that the command would no longer default to stdout if no `--output` flag was specified. I added a fallback here.